sdk/python/kfp/components/structures.py

```diff
@@ -816,14 +816,35 @@ def load_from_component_yaml(cls, component_yaml: str) -> 'ComponentSpec':
             Component spec in the form of V2 ComponentSpec.
         """
 
+        def extract_description(component_yaml: str) -> Union[str, None]:
```

Comment:
I think either way is fine, but i am leaning more towards having it as a helper function in the load_from_component_yaml function, since it a function that would always be called when loading a yaml, so it feels right there. No strong opinion though